openpype/hosts/blender/blender_addon/addons/update_paths_to_last_version.py

```diff
@@ -0,0 +1,213 @@
+                    log.info(f"Alembic file '{previous_name}' updated to '{cache_file.filepath}'.")
+
+                updated_cache_files.append(cache_file)
+
+            # Update modifier's object path
+            parts = modifier.object_path.split("/")
+            parts[-1] = f"{modifier.id_data.name}ShapeDeformed"
+            modifier.object_path = f'{os.path.sep}'.join(parts)
+
+    def get_mesh_sequence_cache_modifiers(self, objects):
+        """Return all MESH_SEQUENCE_CACHE modifiers from a list of objects."""
+        return [mod for obj in objects for mod in obj.modifiers if mod.type == 'MESH_SEQUENCE_CACHE']
+
+    def create_library_override_for(self, collections=None, objects=None, modifiers_cache_files=None):
+        set()  # Use a set to avoid duplicate
+        collections = collections if collections else bpy.context.scene.collection
+        objects = objects if objects else bpy.data.objects
+
+        self.override_collection_and_children(collections)
+
+        # Create library overrides for objects and cache files in a single pass
+        local_object_names = {obj.name for obj in objects if not obj.library}
+        local_cache_names = {cache.name for cache in modifiers_cache_files if not cache.library}
+
+        for obj in objects:
+            if obj.library and obj.name not in local_object_names:
+                obj.override_create(remap_local_usages=True)
+
+        for cache_file in modifiers_cache_files:
+            if cache_file.library and cache_file.name not in local_cache_names:
+                cache_file.make_local()
+
+    def get_alembic_publish_path(self, modifier_object_path):
+        # Get the asset name from the modifier_object_path
+        asset_name = modifier_object_path.split("/")[1].split('_')[0]  # "/" is the blender delimiter
+
+        # Construct the animation directory path
+        anatomy = Anatomy()
+        publish_template = anatomy.templates.get('publish')
+        template_session_data = get_template_data_from_session()
+        template_session_data.update({
+            'root': anatomy.roots,
+            'family': 'animation',
+            'subset': f'{asset_name}_animation',
+        })
+        animation_directory = os.path.normpath(
+            os.path.dirname(StringTemplate.format_template(publish_template['folder'], template_session_data))
+        )
+        if not os.path.exists(animation_directory):
+            return
+
+        # Retrieve the highest version folder from the animation directory
+        last_version = self.detect_higher_version(animation_directory)
+
+        # If no version folder is found, return None
+        if not last_version:
+            return
+
+        # Retrieve the alembic file matching the version
+        version_directory = os.path.join(animation_directory, f'v{last_version}')
+
+        alembic_file = None
+        for abc_file in os.listdir(version_directory):
+            if abc_file.endswith('.abc'):
```

Comment:
extension `.abc` should be put in a global variable at to of the file

Reply:
Fixed in the next commit